os/ProtectedLibs.mk

```diff
@@ -150,6 +150,7 @@ endif
 ifeq ($(CONFIG_AMEBAD_WIFI),y)
 TINYARALIBS += $(LIBRARIES_DIR)$(DELIM)lib_wlan$(LIBEXT)
 TINYARALIBS += $(LIBRARIES_DIR)$(DELIM)lib_wps$(LIBEXT)
+TINYARALIBS += $(LIBRARIES_DIR)$(DELIM)lib_eap$(LIBEXT)
```

Comment:
This means that when loadable, `lib_eap` will be included into the kernel. Then user cannot use this directly.
But in the example, it looks that user tries to use `lib_eap` directly.
Please check it.

Reply:
Dear @jeongchanKim ,
Yes, you are right. But if I don't include the lib here, the loadable_apps would be built fail. May I know how should I deal with it here?
Thank You.

Reply:
You can refer to mm library. There are kmm and umm each for kernel and user side.